scripts/benchmarks/build-benchmarks

```diff
@@ -8,7 +8,7 @@
 
 # generate benchmarks
 echo "generating new benchmarks JSON"
-node scripts/benchmarks/merge-benchmark-data.js | tee benchmarks/2018.json
+node scripts/benchmarks/merge-benchmark-data.js | benchmarks/2018.json
```

Comment:
i really don't see the point in outputting the huge array of new benchmarks to the terminal, it just makes it harder to see any issues.

Reply:
Agreed!